### PR TITLE
Issue 8 vuetify colour theme config

### LINKED ISSUE #8: Vuetify Colour Theme Config
Once the theme colours have been decided in the figma wireframes (https://github.com/codersforcauses/bluecrew/issues/2), the theme colours should be set in the ``main.ts`` file so that we have an easy way to ensure that the developers use consistent colours. See https://vuetifyjs.com/en/features/theme/

### PR DESCRIPTION
**Pull request by yuwengchoong**
## Change Summary
Added vuetify colour themes using the hexcodes from the figma team

- [yes] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [yes] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related issue

- Resolve #8

### CODE REVIEW

File: client/src/main.ts [hunk 1]
```diff
@@ -15,11 +15,25 @@ import router from './router'
 
 const app = createApp(App)
 
+// Object to store colour theme hexcodes
+const colourTheme = {
+  colors: {
+    primary: '#193855',
+    titles: '#D12974',
```
Comment (HuxleyBerry) [inline client/src/main.ts]: Could you give these colours names which give more of an indication of the colour? For example, could the ``titles`` colour be renamed to something like ``primaryPink``?

**Review by HuxleyBerry — CHANGES_REQUESTED** [review]

File: client/src/main.ts (hunk 1 above)
Comment (yuwengchoong) [reply client/src/main.ts]: Yup, I've changed the colour names accordingly

File: client/src/main.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ const app = createApp(App)
 const colourTheme = {
   colors: {
     primaryBlue: '#193855',
-    pink: '#D12974',
+    primaryPink: '#D12974',
     lightBlue: '#3FBEE0',
     creamWhite: '#E9DAC4',
     textGrey: '#4F4F4F',
```
Comment (HuxleyBerry) [inline client/src/main.ts]: Sorry to keep requesting trivial changes, but could you change the name of ``textGrey``? The reason is that when we use this colour in the components, we will be doing something like ``<p class="text-textGrey">Hello</p>``. IMO it looks a little dumb to have "text" twice in a row like that. So can you change "text" to some other word? Some options could be "theme", "custom", or "primary".

**Review by HuxleyBerry — CHANGES_REQUESTED** [review]

File: client/src/main.ts (hunk 2 above)
Comment (yuwengchoong) [reply client/src/main.ts]: All good. I've changed it

**Review by HuxleyBerry — APPROVED** [review]
Nice 👍, lgtm

**Pull request merged**